gulpfile.js

```diff
@@ -13,10 +13,25 @@ var gulp        = require('gulp'),
     minifyHTML  = require('gulp-minify-html'),
     size        = require('gulp-size'),
     imagemin    = require('gulp-imagemin'),
-    pngquant    = require('imagemin-pngquant');
+    pngquant    = require('imagemin-pngquant'),
+		plumber     = require('gulp-plumber'),
```

Comment:
Hey! This is awesome, can you please fix the whitespace and I'll merge? :smiley: